bin/cli.js

```diff
@@ -60,4 +74,10 @@ rl.on('close', () => finish());
 
 rl.on('SIGINT', () => finish());
 
-rl.prompt(true);
+setupHistory(rl, (err, message) => {
+  if (err) {
+    if (message) log(message);
+    if (args.verbose) log(err.stack);
```

Comment:
I'd prefer to use `VERBOSENESS_LEVEL` constants or at least make this explicit i.e. `args.verbose !== 0`.